plat/ti/k3low/common/drivers/pm/include/lpm_handler.h

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
can we check if this is even required? There's no LPM code going in this PR anyway right?

Reply:
Agree, drop this include.